subsys/mgmt/mcumgr/Kconfig

```diff
@@ -297,6 +297,14 @@ endif
 
 endmenu
 
+config MCUMGR_SMP_REASSEMBLY_BT
+	bool "Enable packet reassembly in Bluetooth SMP transport"
+	select MCUMGR_SMP_REASSEMBLY
+	help
+	  When enabled the SMP BT transport will collect all MTU fragments,
```

Comment:
When enabled, the SMP BT transport will buffer and reassemble received partial fragments before passing it for further processing.

Reply:
Done